src/coreclr/src/vm/ceemain.cpp

```diff
@@ -188,7 +188,7 @@
 #include "runtimecallablewrapper.h"
 #include "notifyexternals.h"
 #include "mngstdinterfaces.h"
-#include "rcwwalker.h"
```

Comment:
Is the RCW walker going to be deleted as part of this?

Reply:
That is still TBD. At present the answer is no. The current plan would be to do the removal for .NET 6 or .NET 5.1 instead so we could potentially start removal as soon as the snap from .NET 5 occurs.